docs/setup-cluster/k8s/setup-multiple-resource-managers.rst

```diff
@@ -0,0 +1,167 @@
+.. _multiple-resource-managers:
+
+######################################
+ Configure Multiple Resource Managers
+######################################
+
+.. meta::
+   :description: Discover how to configure and manage multiple resource managers.
+
+.. include:: ../../_shared/attn-enterprise-edition.txt
+
+**********
+ Overview
+**********
+
+Multiple Resource Managers (Multi-RM) for Kubernetes allows you to set up a Determined master
+service in one Kubernetes cluster and schedule workloads in the same or other Kubernetes clusters.
+
+**Resource Pool Relationships**
+
+-  Resource pools have a many-to-one relationship with resource managers.
+-  No single resource pool will span multiple resource managers.
+
+Multiple resource managers are defined in the :ref:`master configuration <master-config-reference>`
+using the ``additional_resource_managers`` and ``resource_pools`` options. Any requests to resource
```

Comment:
To define multiple resource managers, define one default manager and one or many additional managers. Default resource managers are defined in the ``resource_manager`` option, whereas any additional resource managers using the ``additional_resource_manager`` option